charts/s3gw/templates/certificate.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.useCertManager }}
+---
+# s3gw-ca root certificate
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: s3gw-ca-cert
+  namespace: {{ .Values.certManagerNamespace }}
+spec:
+  commonName: s3gw-ca
+  subject:
+    countries:
+    - DE
+    localities:
+    - Nuremberg
+    organizationalUnits:
+    - Aquarist Labs
+    organizations:
+    - SUSE
+    provinces:
+    - Bavaria
+  isCA: true
+  issuerRef:
+    kind: ClusterIssuer
+    name: s3gw-self-signed-issuer
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  secretName: s3gw-ca-root
```

Comment:
I would expect values that suggest that this is not meant to be for production use. Like the "snakeoil" certificates shipped with debian.